src/pages/home/sections/host.js

```diff
@@ -8,7 +8,8 @@ function Host() {
     <section id="host">
       <h2>Host</h2>
       <div className="person-group">
-        <Person name="Kent C. Dodds" twitter="kentcdodds" imgSrc="resources/kentcdodds.png" />
+        <Person name="Kent C. Dodds" twitter="kentcdodds" imgSrc="resources/kentcdodds.png"
```

Comment:
You don't need to change this, but in the future, do you mind formatting things like this (if it wont fit well on a single line):

```javascript
<Person
  name="Kent C. Dodds"
  twitter="kentcdodds"
  imgSrc="resources/kentcdods.png"
  link="https://twitter.com/kentcdodds"
/>
```

Makes things easier to read IMO. Should probably find an ESLint rule to encourage that style.

Like I said. Don't worry about updating this for that. Next person to touch this code can reformat it :-)